client/me/purchases/cancel-purchase/button.jsx

```diff
@@ -368,6 +374,7 @@ export default connect(
 	state => ( {
 		isChatAvailable: isHappychatAvailable( state ),
 		isChatActive: hasActiveHappychatSession( state ),
+		cancellationChatAvailable: isCancellationChatAvailable( state ),
```

Comment:
Based on its purpose, it would be clearer if we name it consistently as `precancellationChatAvailable` and `isPrecancellationChatAvailable()`.